packages/frontend/src/hooks/dashboard/useDashboardChart.ts

```diff
@@ -18,6 +20,7 @@ const useDashboardChart = (tileUuid: string, savedChartUuid: string | null) => {
         invalidateCache,
         granularity,
         isAutoRefresh,
+        context,
```

Comment:
should we also include a similar change to `useDashboardSqlChart`? Since these can be added and scheduled in dashboards?

Reply:
I'll do it in this PR, but first I'll have to refactor query context and move it to common because it has some dependency issues.